boards/nxp/imx943_evk/doc/index.rst

```diff
@@ -375,9 +375,9 @@ For DDR target
 
 Note:
 
-a. Please connect two additional usb2serial converter between Host PC and board's
-auduino interface with dupont cable for M70 in M70 MIX and M71 in M71 MIX.
-Connection as below,
+a. Please connect two additional USB-to-Serial converters between the Host PC and the board's
+Arduino interface using Dupont cables. For M70 in M70 MIX and M71 in M71 MIX,
+make the connections as shown below.
 
```

Comment:
This doesn't belong to this patch. Each patch should be a single logical change. This is a spell fix and needs to be placed in a separate patch.